model/clusters_mgmt/v1/azure_node_pool_type.model

```diff
@@ -75,4 +75,10 @@ struct AzureNodePool {
     // If not specified, Server-Side Encryption (SSE) on the OS Disks of the Nodes of the Node Pool 
     // is performed with platform managed keys.
     OsDiskSseEncryptionSetResourceId String
+
+    // OsDisk contains the configuration for the OS disk used by the nodes in the Node Pool.
```

Comment:
We shouldn't mention OsDisk in the comments because users won't know about the model, they will know about the api attributes defined in the openapi file.

We can omit and just say:

The configuration for the OS disk used by the nodes in the Node Pool